lib/pure/terminal.nim

```diff
@@ -471,11 +471,12 @@ type
     styleBright = 1,     ## bright text
     styleDim,            ## dim text
     styleItalic,         ## italic (or reverse on terminals not supporting)
-    styleUnderscore = 4, ## underscored text
+    styleUnderscore,     ## underscored text
     styleBlink,          ## blinking/bold text
-    styleReverse = 7,    ## reverse
-    styleHidden          ## hidden text
-    styleStrikethrough,  ## strikethrough
+    styleBlinkRapid,     ## rapid blinking/bold text (not widely supported)
```

Comment:
This was added?

Reply:
Yes, that's what exactly is "filling that hole". Earlier the enum was not continuous.